ptvsd/wrapper.py

```diff
@@ -1455,15 +1455,33 @@ def on_ptvsd_systemInfo(self, request, args):
             except AttributeError:
                 impl_desc = 'Python'
 
+        def version_str(v):
+            return '{}.{}.{}{}{}'.format(
+                v.major,
+                v.minor,
+                v.micro,
+                v.releaselevel,
+                v.serial)
+
+        try:
+            impl_name = sys.implementation.name
+        except AttributeError:
+            impl_name = 'Python'
+
+        try:
+            impl_version = version_str(sys.implementation.version)
+        except AttributeError: 
+            impl_version =  version_str(sys.version_info)
+
         sys_info = {
             'ptvsd': {
                 'version': __version__,
             },
             'python': {
-                'version': list(sys.version_info),
+                'version': version_str(sys.version_info),
```

Comment:
The reason why it was an array is to make version checks easier on the receiving side (since you can just take the major or minor version and ignore the rest). Up to you whether that makes more sense or not.

Reply:
On the other side i am using PackageVersion Parser in VS. Since we have to do that anyway for ptvsd version.

Reply:
Using PackageVersion made it easier to compare versions.